### PR TITLE
Only show gene sets in the plots tab when querying gene sets

### PR DESCRIPTION
**Pull request by oplantalech**
# What? Why?
Fix cbioportal/cbioportal#6139.

### CODE REVIEW

File: src/pages/resultsView/plots/PlotsTab.tsx [hunk 1]
```diff
@@ -638,6 +638,12 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                 profiles.map(profile=>profile.molecularAlterationType)
             ).filter(type=>!!dataTypeToDisplayType[type]); // only show profiles of the type we want to show
 
+            // if no gene sets are queried, remove gene set profile from dataTypeIds
+            if (this.props.store.genesets.result!.length == 0) {
```
Comment (alisman) [inline src/pages/resultsView/plots/PlotsTab.tsx]: ===

File: src/pages/resultsView/plots/PlotsTab.tsx [hunk 2]
```diff
@@ -638,6 +638,12 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
...
+                    let index = dataTypeIds.indexOf("GENESET_SCORE");
```
Comment (alisman) [inline src/pages/resultsView/plots/PlotsTab.tsx]: dataTypeIds.includes("GENESET_SCORE")

Comment (oplantalech) [reply src/pages/resultsView/plots/PlotsTab.tsx]: `includes` returns a boolean, how do I delete then the element from the array?

**Comment by oplantalech** [comment]
@alisman Thanks for your comments, I have fixed the issues.

**Comment by inodb** [comment]
@oplantalech 	can u rebase this or give us permissions to change your PR? 

Another thing: it would be good to solve the circleci issue where the hyve jobs are run on your org instead of ours, cause we can't restart these jobs atm

**Pull request merged**